repology/parsers/parsers/repodatadb.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2019 Dmitry Marakasov <amdmi3@amdmi3.ru>
+# Copyright (C) 2019 Jeremiah Mahler <jmmahler@gmail.com>
+#
+# This file is part of repology
+#
+# repology is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# repology is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with repology.  If not, see <http://www.gnu.org/licenses/>.
+
+from typing import Iterable
+
+from repology.packagemaker import NameType, PackageFactory, PackageMaker
+from repology.parsers import Parser
+from repology.parsers.nevra import nevra_construct
+from repology.parsers.sqlite import iter_sqlite
+from repology.parsers.versions import VersionStripper
+from repology.transformer import PackageTransformer
+
+
+class RepodataDbParser(Parser):
```

Comment:
Likewise, let's move this into the same file with `RepodataParser` and name it `RepodataSqliteParser`.